app/controllers/api/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2012 Brian Ploetz (bploetz@gmail.com)
```

Comment:
Your link was a bit wrong, but I got what you mean. Well.. that does not really look alike and you did not copy it, so it's fine.

To summarzie:
* If code was copied and only small changes were made, add attribution
* If code was written by us, maybe inspired, but if it does not look alike, do not add attribution

I guess this is something we can work with.